build/Nuget/Microsoft.VisualStudio.ProjectSystem.nuspec

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2011/08/nuspec.xsd">
+  <metadata>
+    <id>Microsoft.VisualStudio.ProjectSystem</id>
+    <summary>CoreXT package for Microsoft Project System for C# and VB </summary>
+    <description>
+      This Project system is a re-write of the C# and VB project system in Managed Language
+    </description>  
+    <language>en-US</language>
+    <requireLicenseAcceptance>true</requireLicenseAcceptance>
+    <!-- Since VS is the only consumer, now, the version is set to 1.0.0-->
+    <version>1.0.0</version>
+    <authors>Managed Languages</authors>
+    <licenseUrl>http://go.microsoft.com/fwlink/?LinkId=394369</licenseUrl>
+    <projectUrl>https://github.com/dotnet/roslyn-project-system</projectUrl>
+    <releaseNotes>Preview of Roslyn based Project System</releaseNotes>
+    <tags>Roslyn ProjectSystem PS CSharp VisualBasic VisualStudio</tags>
+  </metadata>
+  <files>
+    <file src="Microsoft.VisualStudio.AppDesigner.dll" target="lib\net46" />
```

Comment:
Where's the manifest? Not sure I follow the reason for not adding the Editors/AppDesigner pkgdef? How is this different from the dll's themselves which are also duplicated in VS?